client/src/app/components/node-directory/node-directory.component.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { NodeDirectoryComponent } from './node-directory.component';
+
+describe('NodeDirectoryComponent', () => {
+  let component: NodeDirectoryComponent;
+  let fixture: ComponentFixture<NodeDirectoryComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ NodeDirectoryComponent ]
+    }).compileComponents();
+
+    fixture = TestBed.createComponent(NodeDirectoryComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
```

Comment:
This test currently breaks because of this line. Has something to do with the fact GoJS doesn't play well with Angular's test harness. This was the only workaround I could find
```suggestion
```